demo/demo-spring-boot-discovery/demo-spring-boot-discovery-client/src/test/java/io/servicecomb/demo/discovery/client/DiscoveryClientIT.java

```diff
@@ -0,0 +1,24 @@
+package io.servicecomb.demo.discovery.client;
+
+import org.junit.Before;
+import org.junit.Test;
+import io.servicecomb.demo.TestMgr;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+public class DiscoveryClientIT {
+
+	@Before
+	public void setUp() {
+		TestMgr.errors().clear();
+	}
+
+	@Test
```

Comment:
Written demo/IT code for:
1. To test the discovery of particular micro-service
2. To test the Zuul Proxy gateway feature to redirect the request to proper micro-service.